src/install/krate.rs

```diff
@@ -0,0 +1,24 @@
+use install::Tool;
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct Krate {
```

Comment:
FWIW I typically only use "krate" for the literal name "crate", but other variations like "Crate" I tend to use instead of "Krate" b/c it's valid Rust both ways (only `crate` is invalid)